test/e2e/cluster_config.sh

```diff
@@ -63,6 +63,8 @@ clusters:
     autoscaling_scale_down_enabled: "${autoscaling_scale_down_enabled}"
     ebs_root_volume_size: "550" # required by the limitRanger e2e tests (needs 500Gi ephemoral storage) https://github.com/kubernetes/kubernetes/blob/v1.18.3/test/e2e/scheduling/limit_range.go#L59
     autoscaling_scale_down_enabled: "${autoscaling_scale_down_enabled}"
+    ebs_root_volume_size: "550" # required by the limitRanger e2e tests (needs 500Gi ephemoral storage) https://github.com/kubernetes/kubernetes/blob/v1.18.3/test/e2e/scheduling/limit_range.go#L59
+    autoscaling_scale_down_enabled: "${autoscaling_scale_down_enabled}"
```

Comment:
Seems like I have messed up a merge at some point. Doesn't really matter but a bit confusing.

Reply:
Let's drop it in a separate PR?

Reply:
Sure